### ISSUE TITLE
file system sync does not read `.resource` folder

### ISSUE THREAD

**Issue by frank777777777** [laurent22/joplin#1355]
### Operating system

- macOS
- Linux
- Android

### Application

- Desktop
- Mobile

I am using filesystem syncing, the syncing directory I am using has a `.resources` folder, which is supposed to override `~/.config/joplin-desktop/resources`, is it right?

My problem is, joplin always read `~/.config/joplin-desktop/resources` instead of `.resources` in my sync folder, which causes a problem that I cannot sync attachment between different computers.

For example, I have computer A and computer B, which have their own `~/.config/joplin-desktop/resources`. The image I added from computer A does not appear in computer B, since computer B read its own `~/.config/joplin-desktop/resources` folder.

**Comment by frank777777777** [comment]
The mobile application also cannot read the images, since they are in `~/.config/joplin-desktop/resources` from computers.

**Comment by Dasymi** [comment]
Do you have both computers set to sync using filesystem and pointing to the same folder, that is, a folder that **is not** ~/.config/Joplin-desktop... ?

**Comment by frank777777777** [comment]
> Do you have both computers set to sync using filesystem and pointing to the same folder, that is, a folder that **is not** ~/.config/Joplin-desktop... ?

yes, I have both computers syncing to the same folder other than `~/.config/Joplin-desktop`, and I use nextcloud-desktop to handle the syncing.
The problem is that they read their own '~/.config/Joplin-desktop/resources', so the drag-n-drop function for adding pictures is not useful anymore, since pictures can not be synced to another device.

As for android device, syncing using nextcloud webDAV, I put a '.resources' folder manually in the sync folder, looks like the android app does not read `.resources` as well.
Am I missing any thing?

**Comment by Dasymi** [comment]
> As for android device, I sync using nextcloud webDAV, I put a '.resources' folder manually in the sync folder, looks like the android app refuse to read .resources as well.
I don’t think that will work. The resources in the sync folder have to be put there by Joplin in its own format.

**Comment by frank777777777** [comment]
> > As for android device, I sync using nextcloud webDAV, I put a '.resources' folder manually in the sync folder, looks like the android app refuse to read .resources as well.
> 
> I don’t think that will work. The resources in the sync folder have to be put there by Joplin in its own format.

nah, `resources` is just a bunches files that gets its name referenced. Anyway, local pics I attached from android app, do not get sync to `.resources` either. Therefore whatever format of file Joplin is supposed to put in that folder, it is not putting anything at all.

I found a temporary workaround. I created `resources` in the sync folder, and use symbolic link to `~/.config/Joplin-desktop/`. However, this only works on desktop versions only.
But still, I think `.resources` in sync folder should override `~/.config/Joplin-desktop/` for both platform to work.
